archive/tar/tar_test.go

```diff
@@ -0,0 +1,103 @@
+package tar
```

Comment:
If you're not making use of any private functions or structs, better to define tests as part of package `tar_test` to enforce this.